core/src/main/java/org/infinispan/statetransfer/OutboundTransferTask.java

```diff
@@ -285,8 +286,8 @@ public String toString() {
       return "OutboundTransferTask{" +
             "topologyId=" + topologyId +
             ", destination=" + destination +
-            ", segments=" + segments +
-            ", stateTransferChunkSize=" + stateTransferChunkSize +
+            ", segments=" + new IntSet(segments) +
```

Comment:
Because `IntSet` orders the elements and shortens ranges, e.g. {1, 2, 3, 5} is written as {1-3, 5}.